server/utils.js

```diff
@@ -464,44 +465,47 @@ function dataToSchema(listOfElements) {
 }
 
 function fixAddress(schemaObject, geoObject) {
+  // The returned object will be POST/PATCH(ed),
```

Comment:
then fix is not a proper name, wouldn't it be completeAddress, nonetheless, please add comments they are always helpful